client/ayon_core/plugins/publish/integrate_inputlinks.py

```diff
@@ -9,7 +9,14 @@
 
 
 class IntegrateInputLinksAYON(pyblish.api.ContextPlugin):
-    """Connecting version level dependency links"""
+    """Connecting version level dependency links
+
+    Handles links:
+        - generative - what gets produced from workfile
+        - reference - what was loaded into workfile
+
+    It expects workfile instance is being published.
```

Comment:
So - shouldn't this basically have `families = ["workfile"]` applied? (It won't log the warning message then though)

Reply:
Bit irrelevant to this PR. Not sure why Kuba set it up as ContextPlugin. Maybe because it should handle different links in the future, not depending on workfile instance? (IDK if we have/plan to have such.)

Reply:
Note: `ContextPlugin` can also filter to `families`.